android/src/com/frostwire/android/gui/transfers/TransferManager.java

```diff
@@ -482,7 +484,34 @@ public void resumeResumableTransfers() {
                         continue;
                     }
 
-                    if (bt.isPaused()) {
+                    if (bt.isPaused() && !bt.isFinished()) {
+                        bt.resume();
+                    }
+                } else if (t instanceof HttpDownload) {
+                    // TODO: review this feature taking care of the SD limitations
+                /*if (t.getName().contains("archive.org")) {
+                    if (!t.isComplete() && !((HttpDownload) t).isDownloading()) {
+                        ((HttpDownload) t).start(true);
+                    }
+                }*/
+                }
+            }
+        }
+    }
+
+    public void seedFinishedTransfers() {
+        List<Transfer> transfers = getTransfers();
+
+        if (!isMobileAndDataSavingsOn()) {
```

Comment:
gotta double check this works.

!isMobile == WIFI
Data Savings On == Download only on Wifi.

I think it's better to put a stop condition, or not show the option at all in the UI when the conditions aren't there.